docs/fast_data/architecture.md

```diff
@@ -61,32 +61,84 @@ On the other hand, Kafka and MongoDB can be used in different steps of the proce
 
+The SoRs usually belong to a different network portion, far from all the processing logic, since they must be kept independent of all the Fast Data flow, with the only exception of the CDC being connected to them. 
+
+### Standard Architecture
+
+The standard architecture is rather streamlined, with just a couple of pivot points. A regular flow of information follows this path:
+1. The CDC emits an event stating that some data in the SoR has changed;
+2. The RTU performs the normalization of the messages received by the CDC to select the ones of interest and makes them adhere to a standard of interest, and then stores the Projections on MongoDB;
+3. The RTU computes and emits a `Projection Change` and saves it on MongoDB;
+4. The SVC reads the `Projection Change`by polling MongoDB. Then, it aggregates the Single View using the new data and stores it in MongoDB.
+   1. The SVC emits a `Single View Event` if it is configured to do so. 
+
+![Fast Data Standard Architecture](img/fastdata-architecture-standard.svg)
+
+### Standard Architecture with an SV-Patch
+
+:::info
+Everything related to the Standard Architecture is also valid with an SV-Patch.
+:::
+
+Aggregation is not the only way possible to update Single Views, there is also an alternative called SV-Patch. This kind of operation is strongly recommended when a field of a Projection that is in common with a vast portion of Single Views, is updated. With this operation, the Single View Creator performs a Mongo update starting from the update of a single Projection, without regenerating the whole Single View. For an SV-Patch, the flow of information is as follows:
+1. The CDC emits an event stating that some data in the SoR has changed;
+2. The RTU performs the normalization of the messages received by the CDC to select the ones of interest and makes them adhere to a standard of interest, and then stores the Projections on MongoDB;
+  1. The RTU emits a `Projection Update` event (only for lookup/ constants);
+4. The RTU computes and emits a `Projection Change`  (only for non-lookup/ constants)and saves it on MongoDB;
+5. A second SVC (for SV-Patch operations) consumes the `Projection Update` messages from the RTU then uses them to aggregate the Single View and stores it in MongoDB.
+   1. The SVC emits a `Single View Event` if it is configured to do so. 
+
+![Fast Data Standard Architecture with an SV-Patch](img/fastdata-architecture-standard-svpatch.svg)
+
+Click on [this link](/fast_data/configuration/single_views.md#single-view-patch) for more details on SV-Patch configurations.
+
+### Event-Driven Architecture
+
+:::caution
+The Single View Trigger Generator (SVTG) is still a BETA Plugin and is under active development. Pay attention to using the following Event-Driven Architecture.
```

Comment:
@TheRusCompass Are we going to maintain this banner or should we start removing it?